cfn-stacks/ec2.yaml

```diff
@@ -7,7 +7,48 @@ Parameters:
     Description: Env Suffix for SMBD EC2 Server
     Type: String
 
-Resources: 
+Resources:
+  EC2IAMRole:
```

Comment:
should we move this to the iam template just for organization?

Reply:
@sy556 honestly i just put it there cause i had no idea how to reference resources between templates. is that something that is possible?

Reply:
yea, we're already doing that with the vpc. You can export the arn or any value that you need in the outputs then import it in the other stacks. 

Something like this:
VPC stack:
  SubnetA:
    Description: A reference to the subnet in the 1st Availability Zone
    Value: !Ref SubnetA
    Export:
      Name: !Sub SubnetA${Environment}

EC2 stack:
SubnetId:
  Fn::ImportValue:
    !Sub "SubnetA${Environment}"